application/features/password.feature

```diff
@@ -3,18 +3,17 @@ Feature: Password
   As an authorized user
   I need to be able to update a specific user password
 
-#  Scenario: Provide a new password for an existing user
-#    Given I receive an existing employee id
```

Comment:
Where are the functions for this? Should they not be removed?

Reply:
I removed this for two reasons: it references features that do not exist in id-broker, and it is written like an integration test. In my opinion, unit tests have a higher priority than integration tests, which means that if you have neither, the unit tests should be written first. I would be glad to share more about that if you're interested.